src/sentry/tasks/low_priority_symbolication.py

```diff
@@ -34,7 +34,7 @@ def _scan_for_suspect_projects() -> None:
 
     for project_id in realtime_metrics.projects():
         suspect_projects.add(project_id)
-        update_lpq_eligibility(project_id).apply_async()
+        update_lpq_eligibility.apply_async(kwargs={"project_id": project_id})
```

Comment:
```suggestion
        update_lpq_eligibility.apply_async(project_id=project_id)
```

Isn’t that the goto syntax for passing kwargs?

Reply:
I may have been overthinking this